Provider/src/main/java/com/spotify/confidence/ConfidenceFeatureProvider.kt

```diff
@@ -258,6 +258,10 @@ class ConfidenceFeatureProvider private constructor(
         ): ConfidenceFeatureProvider {
             val configuredClient = client ?: ConfidenceRemoteClient(
                 clientSecret = clientSecret,
+                sdkId = "SDK_ID_KOTLIN_PROVIDER",
+                // x-release-please-start-version
+                sdkVersion = "0.1.2",
+                // x-release-please-end
```

Comment:
This was an interesting approach but I would suggest we use `BuildConfig` instead. 
You can see an example how it is defined [here](https://github.com/spotify/confidence-openfeature-provider-kotlin/blob/main/ConfidenceDemoApp/build.gradle.kts#L41) and used [here](https://github.com/spotify/confidence-openfeature-provider-kotlin/blob/8a1ece8ba4aff7a189f4d10cc38fd279d1d67b27/ConfidenceDemoApp/src/main/java/com/example/confidencedemoapp/ClientSecretProvider.kt#L4).

Reply:
you can also use the inline comment `// x-release-please-version` too